src/openforms/scss/pdfs/_privacy-policy.scss

```diff
@@ -1,11 +1,13 @@
 @use 'microscope-sass/lib/bem';
 
-.submission-privacy-policy {
+.submission-consent {
   display: flex;
   align-items: flex-start;
 
   @include bem.element('text') {
-    padding-left: 5px;
+    flex: 1;
+    padding-left: 8px;
+    min-width: 0;
```

Comment:
could we try `gap: 8px` to get the spacing between elements? Or is there a particular reason why this doesn't work?

So it would become:

```scss
.submission-consent {
  display: flex;
  align-items: flex-start;
  gap: 8px;

  @include bem.element('text') {
    p { ... }
  }
}
```

Reply:
Yes, I used `gap` at the beginning but it doesn't seem to work in the pdf, that's why I left the `padding-left` as it was originally.